components/bl-draggable-list-component/README.md

```diff
@@ -1,44 +1,83 @@
 # Draggable List
 
-The draggable list is a Backendless UI-Builder component. This allows the list to be displayed. List items can also be
-dragged, modified, removed, and added. The component is customizable, you can enable or disable actions with it.
-
-## Usage
-
-### Styles and Settings
-
-<dl>
-<dt>Title</dt>
-<dd>Title for draggable list. Defaults to "List".</dd>
-<dt>Allow Delete</dt>
-<dd>Allows to delete items from the list.</dd>
-<dt>Allow Edit</dt>
-<dd>Allows to change the items in the list.</dd>
-<dt>Allow Add</dt>
-<dd>Allows to add items to the list.</dd>
-<dt>Allow Add</dt>
-<dd>Allows to add items to the list.</dd>
-</dl>
-
-### Event Handlers and Bindable Properties
-
-<dl>
-<dt>Items List Logic</dt>
-<dd>Allows to specify an items list in Backendless Logic.</dd>
-<dt>Title Logic</dt>
-<dd>Allows to specify a title in Backendless Logic.</dd>
-<dt>Allow Delete Logic</dt>
-<dd>Allows to specify permission to delete elements in the Backendless Logic.</dd>
-<dt>Allow Edit Logic</dt>
-<dd>Allows to specify permission to edit elements in the Backendless Logic.</dd>
-<dt>Allow Add Logic</dt>
-<dd>Allows to specify permission to add elements in list in the Backendless Logic.</dd>
-<dt>On Drag Event</dt>
-<dd>Triggered when the user drags list item.</dd>
-<dt>On Delete Event</dt>
-<dd>Triggered when the user delete item from list.</dd>
-<dt>On Edit Event</dt>
-<dd>Triggered when the user edited list item.</dd>
-<dt>On Add Event</dt>
-<dd>Triggered when the user added item to list.</dd>
-</dl>
+This is a component for Backendless [UI-Builder](https://backendless.com/developers/#ui-builder) designer.
+
+This allows the list to be displayed. List items can also be dragged, modified, removed, and added.
+The component is customizable, you can enable or disable actions with it.
+
+<p align="center">
+  <img alt="main thumbnail" height="220" src="./thumbnail.png" width="358"/>
+</p>
+
+## Properties
+
+| Property    | Type       | Default value | Logic              | Data Binding | UI Setting | Description                                                                   |
+|-------------|------------|---------------|--------------------|--------------|------------|-------------------------------------------------------------------------------|
+| itemsList   | *text*     |               | Items List Logic   | YES          | NO         | This is a handler for the logic to determine items list                       |
```

Comment:
If change the type to `JSON` it will work in UI Settings, but I don't sure if that needed